pragma/views/endpoints.py

```diff
@@ -24,12 +24,6 @@ def get(self, oaid=None):
             return OpenAnnotation.get(oaid).dict()
         return {"annotations": [oa.dict() for oa in OpenAnnotation.query.all()]}
 
-    @rest
```

Comment:
It occurs to me there may be changes upstream which have not been pushed to git. I may accept this and then rebase w/ my changes